target/linux/ipq806x/patches-5.10/900-arm-add-cmdline-override.patch

```diff
@@ -1,37 +0,0 @@
---- a/arch/arm/Kconfig
```

Comment:
Is something missing in this commit? You removed patch only...

Reply:
Thx for the report... i notice that cmd override is not enabled in our config but i added it anyway